plugin/hosts/hosts_test.go

```diff
@@ -82,11 +82,11 @@ var hostsTestCases = []test.Case{
 	},
 	{
 		Qname: "example.org.", Qtype: dns.TypeAAAA,
-		Answer: []dns.RR{},
+		Answer: []dns.RR{}, Rcode: dns.RcodeSuccess,
```

Comment:
I would like to see a new test that is failing with the current code and working with the new